docs/userstories.rst

```diff
@@ -35,3 +35,73 @@ During the development, it becomes clear that a major part of the implemented
+development of modules in the workflow system is required.
+A *graphical visualization* of the workflow is not mandatory, but should be very useful to share with and explain the
+workflow to others.
+If possible the individual components of the workflow should be general/modular, such that these are *reusable* by others.
+Others include members of the research group or other researchers in the same field who then quickly can pick up work from
+some point in the pipeline/workflow. While this is not necessary for the paper itself, this is highly desirable to increase 
+efficiency in the research group and / or community.
+
+.. _bamgroup:
+
+BAM Group (structural reliability)
+----------------------------------
+The objective is to guarantee the safety of a construction by determining the failure probability.
+This is done by repeated evaluation of a structural model in a probabilistic approach (*adaptive importance sampling*).
+Herein, an approximation of the probability of failure is computed from a joint distribution function of the
+material parameters. Usually within this framework a large number of evaluations of the structural model are required which is
+why the replacement of the structural model by a surrogate / *reduced order model* is necessary to increase efficiency.
+
+Another aspect of the probabilistic approach is the calibration of the model by means of *Bayesian Inference* in which
+the distribution function is determined based on the prior knowledge and the likelihood function. The likelihood
+describes the probability that the numerical model of the lab experiment reproduces the experimental data for given model parameters.
+Therefore, the numerical model has to be evaluated for various parameter values and the experimental data has to be processed. Here, for each of the different lab experiments a model and corresponding *metadata* should exist, such that these
+are evaluated collectively. At the same time a query of the *database* should extract the relevant metadata and process the raw
+data accordingly.
+
+With the above example we would like to address workflows which consist of several modules, where each module
+requires a different expertise (FEM, ROM, Bayesian Inference, ...), leading to the implementation and development of the
+workflow being a group effort.
+
+Collaborators taking part in the presented workflow could be grouped by *developers* and *users*.
+
+Developers
+^^^^^^^^^^
+It is important that all modules (heterogeneous models) are embedded into a common framework and that the 
+*relations and dependencies* between modules should be defined with a *portable data structure*.
+For collaboration with others within the group automation of the installation of potentially complex
+software environments (up to *machine-independent execution*) within the workflow tool is thought to be a great but
+not compulsory feature.
```

Comment:
why not compulsory, if I want new students to use this tool or having a CI running, then requiring manual installation steps increases significantly the entrance barrier.

Reply:
okay, I didn't think about new students (or colleagues) and agree with that. What I had in mind was rather that each developer installs the needed software locally instead of using a container/docker for development. So in this case the automated installation of software environment is required, but the machine-independent execution is not a compulsory feature.